sympy/logic/algorithms/minisat22_wrapper.py

```diff
@@ -0,0 +1,46 @@
+from sympy.assumptions.cnf import EncodedCNF
+
+def minisat22_satisfiable(expr, all_models=False, minimal=False):
+
+    if not isinstance(expr, EncodedCNF):
+        exprs = EncodedCNF()
+        exprs.add_prop(expr)
+        expr = exprs
+
+    from pysat.solvers import Minisat22
+
+    # Return UNSAT when False (encoded as 0) is present in the CNF
+    if {0} in expr.data:
+        if all_models:
+            return (f for f in [False])
+        return False
+
+    r = Minisat22(expr.data)
+
+    if minimal:
```

Comment:
You are absolutely right! I added them in my latest commit, and pushed the changes. I took this opportunity to merge master into the branch, to avoid conflicts. Somehow I'm not being able to squash that commit with my changes: my attempts to do so make it so that the master changes are reversed.